app/api/__init__.py

```
@@ -2,6 +2,7 @@
 
 from app.api.audio_api import stt_router
 from app.api.audio_services_api import service_router
+from app.api.speaker_api import speaker_router
 from app.api.task_api import task_router
 
-__all__ = ["stt_router", "service_router", "task_router"]
+__all__ = ["stt_router", "service_router", "speaker_router", "task_router"]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)